test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
remove controller tests as we use rspec for tests. remove from rails generator https://guides.rubyonrails.org/generators.html#customizing-your-workflow